bulletroboy/links_mapper.py

```diff
@@ -0,0 +1,12 @@
+import yaml
+
+class LinksMapper(object):
```

Comment:
i honestly do not understand the need to create yet another class for this. the link name mapping can be done in 2 functions. i suggest to move it to the force mapper (merge with the open forces imitator - I also don't know why that one is call imitator, cause it doesn't imitate anything, but transorms/maps the forces) class that does
- link name mapping
- location mapping (e.g. scaling)